libs/designsystem/src/lib/components/segmented-control/segmented-control.component.spec.ts

```diff
@@ -96,6 +99,13 @@ describe('SegmentedControlComponent', () => {
       expect(component.value).toBe(items[2]);
     });
 
+    it('should have touch area with minimum size equal to fat finger size', () => {
+      const element = spectator.element.querySelector('ion-segment-button');
+
+      expect(window.getComputedStyle(element, '::after').minHeight).toEqual(fatFingerSize());
+      expect(window.getComputedStyle(element, '::after').minWidth).toEqual(fatFingerSize());
```

Comment:
This seems to test what the values of the CSS properties are rather than the actual width/height of the pseudo-element. `min-width` and `min-height` won't change depending on the width/height of the content of the element. The computed values of the `width` and `height` properties will.

Consider to test that the width and height of the pseudo-element is greater than or equals `fatFingerSize()` instead:

```
expect(window.getComputedStyle(element, '::after').height).toBeGreaterThanOrEqual(fatFingerSize());
```

Disclaimer: I'm not an expert in writing tests, so you better check that `toBeGreaterThanOrEqual` is a real matcher function 😉

Reply:
Fixed in 44d80b33f8d0b350bdf2f3518eb6013be1bd122b